### ISSUE TITLE
using = to auto-indent a highlighted block of code does not work properly

### ISSUE THREAD

**Issue by val-litvak** [mustache/vim-mustache-handlebars#40]
Before
<img width="268" alt="before" src="https://cloud.githubusercontent.com/assets/767168/8763686/64d58b20-2d6f-11e5-945c-2f462ee351e4.png">

After
<img width="248" alt="after" src="https://cloud.githubusercontent.com/assets/767168/8763687/69bb36c6-2d6f-11e5-88bb-0faa1b72785a.png">

Expected
<img width="211" alt="expected" src="https://cloud.githubusercontent.com/assets/767168/8763688/6efa0c84-2d6f-11e5-80df-a8c171b4fb8a.png">

**Comment by jsit** [comment]
This is working in the latest master for me; can you please test again?

```hbs
<section id="todoapp">
  <header id="header">
    <h1>todos</h1>
  </header>

  {{outlet}}
</section>
```

**Comment by FrancoB411** [comment]
Getting the same issue. Just ran :PlugUpdate
The {{}} all flatten to the left rather than indent. The HTML indenting is fine though.

**Comment by jsit** [comment]
@FrancoB411 What version of Vim are you using? I'm on 8.1.202 and not seeing this issue.

Try running Vim with this simple `.vimrc.mustache` and no other plugins:

```vim
set nocompatible
set runtimepath=~/.vim/bundle/vim-mustache-handlebars,$VIMRUNTIME
filetype plugin on
filetype indent on
syntax on
```

`$ vim -u ~/.vimrc.mustache`

Do you have any HTML plugins that might be conflicting with VIM's native HTML indenting?

**Comment by jsit** [comment]
This may have been fixed thanks to recent PR #52. Please try again. Thanks!
